virtme_ng/utils.py

```diff
@@ -27,3 +29,38 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def get_conf_file_path():
+    """Return virtme-ng main configuration file path."""
+
+    # First check if there is a config file in the user's home config
+    # directory, then check for a single config file in ~/.virtme-ng.conf and
+    # finally check for /etc/virtme-ng.conf. If none of them exist, report an
+    # error and exit.
+    configs = (
+        CONF_FILE,
+        Path(Path.home(), ".virtme-ng.conf"),
+        Path("/etc", "virtme-ng.conf"),
+    )
+    for conf in configs:
+        if conf.exists():
+            return conf
+    return None
+
+
+def get_conf(key_path):
+    conf_path = get_conf_file_path()
+    if conf_path is None:
+        return None
+
+    keys = key_path.split(".")
+
+    with open(conf_path, encoding="utf-8") as conf_fd:
+        conf = json.loads(conf_fd.read())
+        try:
+            for key in keys:
+                conf = conf[key]
+            return conf
+        except (KeyError, TypeError):
+            return []
```

Comment:
Please print  a warning.

Reply:
Added in https://github.com/arighi/virtme-ng/pull/280/commits/b24ab70280e65814e27cf9e877b2f7ac87c0cd7d